src/test/java/com/ft/methodearticleinternalcomponentsmapper/transformation/BodyProcessingFieldTransformerFactoryTest.java

```diff
@@ -1423,6 +1423,13 @@ public void shouldKeepAsideTag(){
         checkTransformation(originalContent, transformedContent);
     }
 
+    @Test
+    public void shouldKeepSTag(){
+        String originalContent = "<p><s>My article is the best</s></p>";
```

Comment:
The `<s>` will be in the `<body>` tag, so update this test accordingly.
Also, I realized the test above, for `<aside>` has this problem also, it should be inside the `<body>` tag.